examples/testbed/ui.rs

```diff
@@ -665,14 +665,14 @@ mod linear_gradient {
                                             angle: LinearGradient::TO_RIGHT,
                                             stops: stops.clone(),
                                         }),
-                                        children![
+                                        children![(
```

Comment:
I feel like this was probably intended to be spawned as a single bundle, as having separate entities with just a `Text` and a `TextFont` respectively seems nonsensical.

If so, then I believe this highlights the value of this PR ;)

Reply:
I think the CI confirms this https://pixel-eagle.com/project/b04f67c0-c054-4a6f-92ec-f599fec2fd1d/run/22845/compare/22830?screenshot=testbed_ui/screenshot-LinearGradient.png (what an amazing setup BTW!). I feel like we probably want to use different font settings as this makes the output less readable.